safe-global/safe-homepage
# feat: Add missing blocks to ecosystem page

## What it solves

Part of #176 

Missing links can be added later

## How to test

1. Open `/ecosystem`
2. Observe 2 blocks from the Design being there
3. Test it on mobile as well

## Screenshots

<img width="1266" alt="Screenshot 2023-05-09 at 12 47 35" src="https://github.com/safe-global/safe-homepage/assets/5880855/aa69dcad-8198-462c-bbda-8697c3c07024">
<img width="517" alt="Screenshot 2023-05-09 at 12 47 46" src="https://github.com/safe-global/safe-homepage/assets/5880855/aad727f3-2dd9-4527-8e1b-a0825a84c9fc">
<img width="1440" alt="Screenshot 2023-05-09 at 12 47 58" src="https://github.com/safe-global/safe-homepage/assets/5880855/198c2338-b809-443b-8d5f-b2ca2be378ee">
<img width="518" alt="Screenshot 2023-05-09 at 12 48 11" src="https://github.com/safe-global/safe-homepage/assets/5880855/4eea0364-95ae-4009-90a3-2bbe36ff2dcc">
